hack/get_ocp_versions_for_testing.sh

```diff
@@ -2,7 +2,8 @@
 
 set -euo pipefail
 
-SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+CURRENT_FILE=$(python -c "import os; print(os.path.realpath('$0'))")
```

Comment:
```suggestion
SCRIPT_DIR=$(python -c "import os; print(os.path.dirname('$0'))")
```

Reply:
is it resolve the relative path?

Reply:
Not sure which flow takes that into account, but from what I saw it produced the same result.
Can you point to a case where it fails?

Reply:
ok. I tested it and the dirname function does not resolve the relative path.
For example I was at the /Users/michaellevy/go/assisted-installer-agent and run the command:
python -c "import os; print(os.path.dirname('..'))" 
and it returns an empty string instead of /Users/michaellevy/go

Reply:
The original code uses readlink -f which resolve all the symlinks and relative path to the absolute path.
To be on the safe side I used a similar functionality to cover all the edge cases since my goal here is to support macOS.